### ISSUE TITLE
Opening url with special characters doesn't work correctly on wsl

### ISSUE THREAD

**Issue by SolomonRosemite** [amodm/webbrowser-rs#73]
When opening a browser with the following url `https://accounts.spotify.com/authorize?response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Fcallback&client_id=x&state=y&scope=user-read-recently-played
`  the browser opens successfully but instead of landing on the expected url, the browser lands here `https://accounts.spotify.com/authorize?response_type=code`.
Cutting off everything after the "&" character.

When trying to reproduce this using the command from here:
https://github.com/amodm/webbrowser-rs/blob/322bc6132072368cf0ef96bd9217762c64f06514/src/unix.rs#L198

I get the same result and the following output:
```
C:\Users\solomon>start https://accounts.spotify.com/authorize?response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Fcallback&client_id=x&state=y&scope=user-read-recently-played
'redirect_uri' is not recognized as an internal or external command,
operable program or batch file.
'client_id' is not recognized as an internal or external command,
operable program or batch file.
'state' is not recognized as an internal or external command,
operable program or batch file.
'scope' is not recognized as an internal or external command,
operable program or batch file.
```

However, with the help of [this](https://superuser.com/a/36730), changing the command to the following seemed to work for me:
```
C:\Users\solomon>start "" "https://accounts.spotify.com/authorize?response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Fcallback&client_id=x&state=y&scope=user-read-recently-played"
```

Let me know if any more info needed.

**Comment by amodm** [comment]
Can you try once with the code in the fix/issue-73 branch? https://github.com/amodm/webbrowser-rs/pull/74/commits/3da3e74aa2a4001e84899dd12544b7554e8b95b6

It seems to work in my WSL sandbox, but would prefer your verification before merging this to main.

**Comment by amodm** [comment]
@SolomonRosemite I'll wait a couple of days to hear from you, else I'll assume this fix (and my testing) to be sufficient.

**Comment by SolomonRosemite** [comment]
Seems to work perfectly. Thanks!

**Comment by amodm** [comment]
Awesome, thanks for the quick confirmation! This is now released as [v0.8.11](https://github.com/amodm/webbrowser-rs/releases/tag/v0.8.11)
